src/polymerday-sponsors-list.html

```diff
@@ -36,13 +76,31 @@
       is: 'polymerday-sponsors-list',
 
       properties: {
-        sponsors: Array,
-        sponsorsType: String,
-        sponsorsTitle: String
+
+        sponsorType: String,
+
+        _title: {
+          type: String,
+          computed: '_computedTitle(sponsorType)'
+        },
+
+        _titlesMap: {
```

Comment:
This create a dependency, you don't know need to know the sponsor types. It should be a property